~

```diff
@@ -0,0 +1,54 @@
+# This is a combination of 3 commits.
```

Comment:
@eohartman can you also please remove this file named `~` from your feature branch?

I thought that was something that I inadvertently added locally in my test and then removed, but I've just noticed that it features in your PR too. Running the following from the top level directory in your feature branch should remove that file:

```
$ rm \~
```

Thanks!